demo/main3.html

```diff
@@ -184,11 +180,17 @@ <h2 class="section-title">Contact Us</h2>
 </footer>
 <script src="http://localhost:8080/api/web-nexus"></script>
 <script>
-  webNexus.setChatBoxDivId('chat-container')
-  webNexus.createLoginForm();
-  document.getElementById('new-chat-button').addEventListener('click', function () {
-    webNexus.createChatBox(document.getElementById('chat-recipient-input').value.trim());
-  });
+    const onSuccessfulLogin = (data) => {
+        const url = 'http://localhost:63342/web-nexus/sdk/pages/demo-page.html';
+        document.cookie = data.accessTokenCookie;
```

Comment:
What does this do?

Reply:
Demo callback, which have to be executed after successful login or register.

Reply:
No, i mean what is document.cookie? What does it do?